benchmarks/gabe-fs-mdx/gatsby-config.js

```diff
@@ -13,8 +15,11 @@ module.exports = {
     {
       resolve: `gatsby-plugin-mdx`,
       options: {
-        lessBabel: true,
+        defaultLayouts: {
+          articles: path.resolve(`./src/templates/blog-post.js`),
```

Comment:
I added this for testing. The old benchmark did not use this file at all. See changes in `gen.js`.